tests/test_config.py

```diff
@@ -0,0 +1,26 @@
+import os
+import pytest
+
+from kpconv_torch.utils.config import valid_dataset, load_config, save_config
+
+
+def test_valid_dataset():
+    """
+    If the dataset is supported, valid_dataset returns it, otherwise it raises a ValueError.
+    """
+    assert valid_dataset("S3DIS") == "S3DIS"
+    with pytest.raises(ValueError):
+        valid_dataset("wrong_dataset")
+
+
+def test_load_config():
+    assert load_config("tests/config_S3DIS.yml")["dataset"] == "S3DIS"
```

Comment:
You may increase (to be verified) the coverage statistics by checking the values of other config variables.

Reply:
Done. We will see.